src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/TaskCompleteNotification.java

```diff
@@ -0,0 +1,32 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getAndReplaceText;
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+
+public class TaskCompleteNotification extends Notification {
+
+  /**
+   * Constructor.
+   */
+  public TaskCompleteNotification(String contentKey, int index) {
```

Comment:
Make the constructor private, so that one can only create this notification through these factory methods below